suites/pacific/cephfs/tier-4_fs.yaml

```diff
@@ -0,0 +1,149 @@
+---
+#=======================================================================================================================
+# Tier-level: 2
```

Comment:
Likewise, could we have the name of the test suite identifying it's purpose?

Reply:
Sure